doc/0000-ccl_note/main.tex

```diff
@@ -764,25 +849,32 @@ \subsubsection*{Standard tracers}
 computed assuming a linear-theory relation between the matter overdensity and
 peculiar velocity fields. While this should not be problematic for wide photometric
 redshift bins, users should exercise care when interpreting results for narrow
-window functions. For intrinsic alignments, (I.A. in the table), \ccl currently
-supports the so-called ``non-linear alignment model'', according to which the
+window functions.
+
+By default, for all of these tracers, \ccl assumes that the underlying power spectrum
+is given by $P_{\rm NL}$, which is then multiplied by the relevant (linear) biasing.
+The functionality of these tracers can be used with more general power spectra $P_{\alpha\beta}$,
+e.g.\ the nonlinear biasing or IA described in Sec.~\ref{sec:Nonlinear_Pk}. In this case,
+the biasing should be set to 1 when defining these tracers instead specified in the relevant {\tt PTTracer}.
+
+For intrinsic alignments, (I.A. in the table), the default \ccl
+choice is the ``non-linear alignment model'', according to which the
 galaxy inertia tensor is proportional the local tidal tensor
 \cite{2004PhRvD..70f3526H,2007MNRAS.381.1197H}. $\chi_s$ is the distance to
 the source plane for CMB lensing. $b_{\rm IA}$ is the corresponding alignment
 bias. Notice that in the \ccl implementation, the normalization of this bias
```

Comment:
I would mention the two normalization options you introduced.

Reply:
This is done. Let me know if you approve.